pkg/api/componentreadiness/query/querygenerators.go

```diff
@@ -405,12 +408,14 @@ func getTestDetailsQuery(
 						ANY_VALUE(cm.jira_component) AS jira_component,
 						ANY_VALUE(cm.jira_component_id) AS jira_component_id,
 						COUNT(*) AS total_count,
+						ANY_VALUE(jobs.prowjob_url) AS prowjob_url,
```

Comment:
ANY_VALUE good here? We're not doing a group by so it looks ok, but caught my eye, I can't recall why it's used here at all...

Reply:
I just followed the other examples here.  This query could maybe be fixed up to avoid it with group bys but as is bigquery complains, even though there's only one option for prowjob_url.  The links are showing up correctly for me.

```
Error: Return code = 500 (error querying component test details from big query: [googleapi: Error 400: SELECT list expression references jobs.prowjob_url which is neither grouped nor aggregated at [17:49], invalidQuery])
```